src/pages/auth/profile/index.tsx

```diff
@@ -1,94 +1,203 @@
-              <>
-                <strong>Profile image:</strong>
-                <Image
-                  alt="profile image"
-                  loader={() => userImage}
-                  src={userImage}
-                  width={200}
-                  height={200}
-                />
-                <br />
-              </>
+        <div className={styles.profileDetailsWrapper}>
+          <h3>Info</h3>
+          <div className={styles.profileDetails}>
+            {userImg ? (
+              <Image
+                alt="profile image"
+                loader={() => userImg}
+                src={userImg}
+                width={200}
+                height={200}
+              />
+            ) : (
+              <div className={styles.imgPlaceholder}></div>
             )}
-          </p>
-
-          <h2>Reset password</h2>
-          <form>
-            <input
-              type="password"
-              placeholder="Old password"
-              value={oldPass}
-              onChange={e => setOldPass(e.target.value)}
-            />
-            <input
-              type="password"
-              placeholder="New password"
-              minLength={passwordMinLength}
-              value={password}
-              onChange={e => setPassword(e.target.value)}
-            />
-            <input
-              type="password"
-              placeholder="Confirm new password"
-              minLength={passwordMinLength}
-              value={confirmPass}
-              onChange={e => setConfirmPass(e.target.value)}
-            />
-            <button type="button" onClick={handleChangePassword}>
-              Reset password
-            </button>
-          </form>
-        </>
+            <div className={styles.profileDetailsText}>
+              <div className={styles.profileDetailsInfo}>
+                <span>Email:</span>
+                <div>{userEmail}</div>
+              </div>
+              <div className={styles.profileDetailsInfo}>
+                <span>Username:</span>
+                <Input
+                  type="text"
+                  id="username"
+                  placeholder={username || 'Enter username...'}
+                  required={false}
+                  setInputVal={setNewUserName}
+                />
+                <Button
+                  onClick={async () => await handleUpdateUsername()}
+                  style="primary"
+                  content="Update"
+                  arrowDirection="right"
+                />
+              </div>
+              <div className={styles.profileDetailsInfo}>
+                <span>Birth year:</span>
+                {/* TODO: Load birthyear */}
+                <Input
+                  type="number"
+                  id="birthyear"
+                  placeholder={''}
+                  required={false}
+                />
+                <Button
+                  onClick={() => toast('Place holder for update birthyear')}
```

Comment:
Should say like "Coming soon!" since it's possible a user will see this. Leave todo comments and dev notes for actual TODO comments

Reply:
Sure!